src/main/groovy/com/excelsiorjet/gradle/plugin/ExcelsiorJetPlugin.groovy

```diff
@@ -123,13 +123,8 @@ class ExcelsiorJetPlugin implements Plugin<Project> {
             }
         }
 
-        extension.conventionMapping.appType = {
-            if (isWar)  {
-                return ApplicationType.TOMCAT.toString()
-            } else {
-                return ApplicationType.PLAIN.toString();
-            }
-        }
```

Comment:
The same applies here. This is constant assignment as isWar is evaluated when plugin is applied.